EPN article (#71)
virtualscienceforum/virtualscienceforum: texts/epn_paper.md

> # Let's talk online

Author: That's a tentative title, opening a discussion of the final one.

Author: This title I OK, but if anyone has better idea, please add.

Reviewer: This is similar, but:
```suggestion
# Let's take this discussion online
```
or 
```suggestion
# Let's continue this discussion online
```

Reviewer 2: I like Evert's suggestion.

> 

Author: Add your text directly below.

**Important:** before starting to write a paragraph, leave a comment in the paragraph title, so that others don't work on it as well.

> #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk.

Author: Taking this one.

# Let's talk online

#### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk.

Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home—hopefully safe and healthy. Your classes are now online, and probably you had to quickly figure out how run them at all. If you do experiments, your lab is either closed or operates at a much lower capacity. Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself. The only difference is that it happened online.

#### Online talks were gaining momentum already before, and for a good reason.

> > List some of those reasons

Reviewer 3: Starting this paragraph

#### Online talks were gaining momentum already before, and for a good reason.
> List some of those reasons

#### We decided to develop online conferences and established VSF.

#### In thinking about online events we had to revisit many assumptions about conferences.

As we started to organize ourselves, it became clear quickly that in order to achieve our goals, we needed to revisit our assumptions about online events. For example, the interactions between the speaker and the audience as well as among the aduence are quite different and much less spontaneous in an online event. The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions. Likewise, the length of an online event is constraint by very different factors, such as the spread of timezones and local commitments and distractions of the participants.

> #### We ran a couple of events, and saw that they work well.

Reviewer 4: Starting this paragraph on "our experience"

# Let's talk online

#### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk.

Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home—hopefully safe and healthy. Your classes are now online, and probably you had to quickly figure out how run them at all. If you do experiments, your lab is either closed or operates at a much lower capacity. Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself. The only difference is that it happened online.

#### Online talks were gaining momentum already before, and for a good reason.
Online events were gaining momentum even before the crisis hit.
There are several advantages of virtual conferences over conventional, in-person gatherings.
With the increasing concerns about greenhouse emissions, many prefer limiting long-distance travel.
For those with limited mobility, either for family or health reasons, they are much easier to attend.

> Most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone, allowing faster turnaround in disseminating new results.

Author: Leaving a note for later: here we should also mention accessibility to broader community.

Author: done now

> #### We decided to develop online conferences and established VSF.

Author: I'll write this one.

# Let's talk online

<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->

Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.

> Your classes are now online, and probably you had to quickly figure out how run them at all.

Reviewer: Your classes are now taught online, and you've probably had to quickly figure out how run them at all.

If you do experiments, your lab is either closed or operates at a much lower capacity.
Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself.

> The only difference is that it happened online.

Reviewer: The only difference is that it happened online, perhaps as part of a virtual conference.

<!-- #### Online talks were gaining momentum already before, and for a good reason. -->

> Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.

Reviewer: Virtual conferences were gaining momentum before the coronavirus crisis because of multiple advantages over traditional in-person gatherings.

With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.

> To some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.

Reviewer: for -> due to?

Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.

> They are also more inclusive and reach a broader audience especially if the talks are recorded.

Reviewer: They are also more inclusive and reach a broader audience, especially if the talks are recorded.

<!-- #### We decided to develop online conferences and established VSF. -->

> After finding each other as like-minded individuals with an urge to improve the present situation, we started gathering in the late summer of 2019 to explore our options. 

Reviewer: 'present situation' doesn't fit well with the previous paragraph, in my opinion.
What about something more like "individuals who recognized the potential of virtual conferences"?

Author: See also my proposal below

> After finding each other as like-minded individuals with an urge to improve the present situation, we started gathering in the late summer of 2019 to explore our options. 

Author: Here I think "we" comes out of nowhere, and the connection to the previous paragraph is unclear.

How about
```suggestion
The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researhers.
We started exploring the opportunities and challenges of online conferences in late Summer 2019—well before the COVID pandemic.
We decided to keep our organization open for anyone to join, and all of our documentation and notes public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
```

Reviewer: I agree that the 'we' needs a bit more clarification. I like your suggestion too.

# Let's talk online

<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->

> Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.

Reviewer 2: Your routine changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak. You now mostly stay at home, hopefully safe and healthy.

To some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.
Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.
They are also more inclusive and reach a broader audience especially if the talks are recorded.

![](planning_checklist.png)

**Figure caption:** A conference organizer checklist.

<!-- #### We decided to develop online conferences and established VSF. -->

After finding each other as like-minded individuals with an urge to improve the present situation, we started gathering in the late summer of 2019 to explore our options. 
It quickly became clear that we had the capacity to establish a venue for virtual conferencing. 
So, we committed ourselves to develop this idea into a real platform, which we now call the Virtual Science Forum.

<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->

> As we started to organize ourselves, it became clear quickly that in order to achieve our goals, we needed to revisit our assumptions about online events.

Reviewer: it quickly became clear (?)

For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions.
Likewise, the length of an online event is constrained by very different factors, such as the spread of timezones and local commitments and distractions of the participants.

<!-- #### We ran a couple of events, and saw that they work well. -->

We created toolkits and developed strategies and guidelines to navigate these challenges.

> And we got to work to organize online events, starting with a beta test workshop in December 2019.

Reviewer: To navigate these challenges we built toolkits, developed strategies, and created guidelines.
We got to work on organizing our first online test workshop in December 2019, putting our guidelines and strategies to good use.

Author: Fixed in the latest version

Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
You now mostly stay at home, hopefully safe and healthy.
Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
If you do experiments, your lab is either closed or operates at a much lower capacity.
Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or have even given one yourself.
The only difference is that it happened online, perhaps as part of a virtual conference.

<!-- #### Online talks were gaining momentum already before, and for a good reason. -->

Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and online events are the only available option.

> Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.

Author: ```suggestion
Finally, online events are free from most of the expensive and time-consuming aspects of conference organisation: finding a venue, arranging lodging and catering for the participants.
```

This is to change the awkward "gone entirely".

> With planning and logistics out of the way, virtual conferences can be organized much faster and respond to the latest developments.

Author: ```suggestion
With planning and logistics out of the way, virtual conferences can be organized much faster and respond to the latest developments—see a typical timeline in the figure.
```

Reviewer: Reference the figure

> They are also more inclusive and reach a broader audience especially if the talks are recorded.

Author: ```suggestion
Because of being easier to attend and easily accommodating hundreds of participants, online events are more inclusive, and reach an even broader audience if the talks are recorded.
```

![](planning_checklist.png)

**Figure caption:** A conference organizer checklist.

<!-- #### We decided to develop online conferences and established VSF. -->

The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.
We started exploring the opportunities and challenges of online conferences in late Summer 2019, well before the COVID-19 pandemic.
We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
After several discussions we committed to developing a community platform for online conferences, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).

<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->

> Once we started brainstorming online events, we realized, that despite those are similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.

Reviewer: ```suggestion
Once we started brainstorming about online events, we realized that, despite them being similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.
```
Could someone have another look at this sentence? I stared at it for a bit and am not sure what is bothering me actually. Maybe a mismatch between online events and conferences?

Reviewer 5: My feeling is that we aim to mention that the goals are the same but implementations are largely different.

Suggestion:
Once we started brainstorming about online scientific events, we realized that, despite having the same goal its offline counter-parts, their implementations largely differ.

Author: Yep, "them" could refer to both "online events" and "conferences". Probably better to rewrite even more.

```suggestion
Once we started brainstorming how to organize online events, we realized that a lot of usual assumptions about scientific meetings do not apply anymore.
Already the scales of the events are very different: while a typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract more than 500 attendees.
```

> Once we started brainstorming online events, we realized that some of our typical assumptions about conferences do not hold in a virtual format.

Author: ```suggestion
Once we started brainstorming how to organize online events, we realized that a lot of usual assumptions about scientific meetings do not apply anymore.
Already the sizes of the events are very different: while a typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract more than 500 attendees.
The duration of an online event is limited to a few hours because it needs to fit into the daily routines of participants spread across multiple hour zones—this in contrast to the median duration of a conference being a full working week.
```

With more and more people getting used to online meetings and seminars -- and with online conferencing software continually improving -- we expect and hope that online events keep growing. 
Not because of our platform; but because they allow for broad, easy accessibility and because of their potential contribution to reducing carbon emission.
Our effort towards online conferences was not the first and is certainly no longer one of few.
Online colloquia and seminars are organized by many science departments and research institutes.
Virtual workshops and conferences are also on the rise, although at a lower rate due to requiring more coordination and planning.
We also expect to see more organizations like [researchseminars.org](https://researchseminars.org) that distribute announcements in order to help navigate the rapidly growing volume of online events.

<!-- #### You should definitely get involved in the online events, and we are here to help. -->

Online conferencing is emerging right now because there are no alternatives until the pandemic is over, lockdowns are lifted, and travel is restored.

> While our hands are forced now, we hope that the research community will grow accustomed to this long overdue change and will see the value of virtual conferences in their own right.

Reviewer: ```suggestion
While our hands are forced, we hope that the scientific community will grow accustomed to this long overdue change and will see the value of virtual conferences in their own right.
```
Removed 'now' because of previous sentence. Scientific community instead of research, broader?

# Let's talk online

> 

Author: I would like an explicit approval from everyone listed: @xabomon @vfatemi @greplova @everthemore @babaks77 @dvarjas.

```suggestion
## Authors (in alphabetical order)

- Anton Akhmerov (Delft University of Technology)
- Xavier Bonet-Monroig (Leiden University)
- Valla Fatemi (Yale University)
- Eliska Greplova (ETH Zürich)
- Evert van Nieuwenburg (University of Copenhagen)
- Babak Seradjeh (Indiana University)
- Daniel Varjas (Delft University of Technology)
```

Reviewer: 👍

Reviewer 5: Sorry for my little contribution on the writing. I'm OK with either being included or not.
Thanks for the effort, it looks great!

Reviewer 2: 👍

Author: Still waiting for @vfatemi @dvarjas @babaks77

Reviewer 3: :+1:

Reviewer 4: Approved

<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->

Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
You now mostly stay at home, hopefully safe and healthy.
Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
If you do experiments, your lab is either closed or operates at a much lower capacity.

> Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even gave one yourself.

Author: ```suggestion
Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even given one yourself.
```

Reviewer: I think you're missing a 'have' here -> likely attended a scientific presentation or have even given one yourself

The only difference is that it happened online, perhaps as part of a virtual conference.

<!-- #### Online talks were gaining momentum already before, and for a good reason. -->

Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.

> For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.

Author: ```suggestion
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and the online events are the only available option.
```

Reviewer: ```suggestion
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether and online events are the only available option.
```

Author: This is a composite sentence. IIRC those always need a comma.

Reviewer: Oh sorry, I meant to only remove the 'the'. Thanks for spotting that

Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.

> With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.

Author: ```suggestion
With planning and logistics out of the way, virtual conferences can be organized much faster and respond to the latest developments.
```

They are also more inclusive and reach a broader audience especially if the talks are recorded.

![](planning_checklist.png)

**Figure caption:** A conference organizer checklist.

<!-- #### We decided to develop online conferences and established VSF. -->

The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.

> We started exploring the opportunities and challenges of online conferences in late Summer 2019—well before the COVID-19 pandemic.

Reviewer: ```suggestion
We started exploring the opportunities and challenges of online conferences in late Summer 2019, well before the COVID-19 pandemic.
```
Style change suggestion, the previous scentence is 'already' using a dash.

We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.

> It quickly became clear that we had the capacity to establish a venue for virtual conferencing. 

Author: ```suggestion
```

> So, we committed ourselves to developing this idea into a real platform, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).

Author: ```suggestion
After several discussions we committed to developing a community platform for online conferences, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).
```

<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->

> As we started to organize ourselves, we realized right away that in order to achieve our goals, we needed to revisit our assumptions about conferences.

Author: ```suggestion
Once we started brainstorming online events, we realized despite being similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.
```

> For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.

Author: I think this deserves elaboration: "quite different" how? Why "less spontaneous"?

Reviewer: ```suggestion
For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 
As a speaker, for example, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 
This in turn makes setting the atmosphere more difficult, not knowing whether potential jokes were a success or not.
```
Something along these lines, as a suggestion

Author: I'll add this as a start, and iterate later

Reviewer: I'm happy to go over things again later today; group meeting atm

The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions.

> Likewise, the length of an online event is constrained by very different factors, such as the spread of timezones, local commitments and distractions of the participants.

Author: Perhaps so:

```suggestion
Likewise, the length of an online event is constrained by daily routine of the attendees spread across multiple hour zones, and not the need to accommodate travel of many participants to a single location.
```

Reviewer 2: ```suggestion
Likewise, the length of an online event is constrained by the daily routine of the attendees spread across multiple hour zones. On the other hand, other practical issues like the need to accommodate travel of many participants to a single location is not constraining to an online event.
```

Author: I propose to move this to the beginning of the paragraph: the difference in scales is much easier to start with.

<!-- #### We ran a couple of events, and saw that they work well. -->

> We created toolkits and developed strategies and guidelines to navigate these challenges.

Author: Not sure what we mean as "toolkits" especially in plural. What about something more specific:

```suggestion
To guide the community members, we wrote basic guidelines for all online event participants: organizers, moderators, and the audience.
```

Reviewer: Should we maybe include a sentence somewhere earlier where we introduce the concept of a moderator?

Reviewer 5: Indeed, "toolkit" implies to me that we developed some software or something similar. This change is more accurate to what you've done.

Author: @everthemore indeed, I still want to work on the paragraph about difference. It will benefit from extra detail.

> And we got to work to organize online events, starting with a beta test workshop in December 2019.

Author: ```suggestion
Once we felt sufficiently confident, we ran an trial workshop with our direct colleagues as both speakers and the audience.
```

> We learned in real time about challenges of coordinating live questions, addressing technical glitches, and giving feedback to the audience and the speaker.

Author: Here the "in real time" qualifier is unclear—do we want to emphasize that we were gaining experience as the event went on? Is that information important?

```suggestion
We used this test run to try out different formats of accommodating interaction between the audience and the speaker, check out unexpected technical problems, and to collect the impressions from a broader group of colleagues.
```

> More importantly, however, we were impressed by the quality of the event and the possibility of creative ways of engagement in an online setting.

Author: "quality of the event" reads like self-praise.

```suggestion
More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
```

> For example, "watching parties" with a number of participants attending as a group was effective in producing local discussions that were then shared online. And one of our speakers guided the audience through the technical parts of their talk using live code.

Author: ```suggestion
Some of our attendees planned their own "watching parties" with a number of participants viewing as a group that pre-discussed their questions and afterwards shared them with the speaker.
Overall we received overwhelmingly positive feedback from all participants.
```

An attentive reader will notice that also conventional presentations easily include live coding.

> With this foundation in place, we were ready to hit the ground running and organized an inaugural event when the APS March Meeting was cancelled in March 2020 due to the pandemic.

Author: ```suggestion
With this experience, we were ready to hit the ground running and managed to organize a virtual session within a only of days when the APS March Meeting was cancelled.
```

Reviewer: ```suggestion
With this experience, we were ready to hit the ground running and managed to organize a virtual session within only a few days after the APS March Meeting was cancelled.
```
fixed some typo's

Reviewer 5: Seems to be a typo in the text *within a ~~only of~~ few days*

We are now organizing the biweekly VSF Long Range Colloquium series.

> We are also excited to have our first conference application already, and look forward to engaging more community initiatives with our platform.

Reviewer: I feel like up to this point the 'conference application' comes a bit out of the blue. 
```suggestion
We also actively welcome community initiatives for organizing virtual conferences using our platform, and are excited to already have the first of such conference applications in progress.
```

> Our experience shows that, with a fresh perspective, online conferences can work well to bring researchers together and communicate their work effectively.

Author: ```suggestion
Our experience shows that, with a fresh perspective, online conferences work well to bring researchers together and let researchers share and discuss their work effectively.
```

Reviewer 2: ```suggestion
Our experience with organizing online conferences and colloquia has shown that they provide a stimulating environment and foster fruitful discussions.
```

I just removed a comma.

<!-- #### We are confident that we will see an explosive growth of online events. -->

> Our experience with organizing different types of events, such as online conferences and colloquia, has shown that they provide stimulating environments filled with interesting talks and fruitful discussions.

Author: ```suggestion
Our experience with organizing different types of events, such as online conferences and colloquia, has shown that they provide a stimulating environment and foster fruitful discussions.
```

> Adding their broad and easy accessibility, improved software options and overall emphasis on lowering carbon emissions, we can only expect further growth of online events.

Author: Struggling with this sentence: it combines conceptually very different items: "their accessibility", "software options", and "emphasis" (whose emphasis?).

Reviewer: ```suggestion
With more and more people getting used to online meetings and seminars -- and with online conferencing software continually improving -- we expect and hope that online events keep growing. 
Not because of our platform; but because they allow for broad, easy accessibility and because of their potential contribution to reducing carbon emission.
```

While the recent public health situation has pushed us toward creating new avenues to establish effective online communications, the advantages and engagement we are seeing go well beyond the crisis situation.

> Broad adoption of online events will lead to increased interconnectedness and accessibility of science in general.

Author: These don't seem relevant to the message of the paragraph. Are you sure we need those?

> 
> <!-- #### Because of how easy it is, many are working towards this goal. -->
> 

Author: The paragraph below fits well with the one above.

```suggestion
```

> Our effort towards online conferences is not the first and now by far not the only one.

Reviewer: ```suggestion
Our effort towards online conferences was not the first and is certainly no longer one of few.
```

Online colloquia and seminars are organized by many science departments and research institutes.

> Virtual workshops and conferences are also on the rise, although at a lower rate due to requiring more coordination and planning.

Reviewer: This could be a good place to silently hint towards our guides again?

We also expect to see more organizations like [researchseminars.org](https://researchseminars.org) that distribute announcements in order to help navigate the rapidly growing volume of online events.

<!-- #### You should definitely get involved in the online events, and we are here to help. -->

> The domain of online conferencing is emerging *right now* because there are no alternatives until the pandemic is over, lockdowns are lifted, and travel is restored.

Author: Do we need the emphasis in *right now*? The next sentence also reiterates the same point, but explicitly.

```suggestion
Online conferencing is emerging right now because there are no alternatives until the pandemic is over, lockdowns are lifted, and travel is restored.
```

# Let's take this discussion online

## Authors (in alphabetical order)

- Anton Akhmerov (Delft University of Technology)
- Xavier Bonet-Monroig (Leiden University)
- Valla Fatemi (Yale University)
- Eliska Greplova (ETH Zürich)
- Evert van Nieuwenburg (University of Copenhagen)
- Babak Seradjeh (Indiana University)
- Daniel Varjas (Delft University of Technology)

## Introduction

> With the lockdowns caused by the COVID-19 pandemic researchers turn to online conferencing. While posing new challenges, this format also brings multiple advantages. We argue that virtual conferences will become a common way of scientific communication, and invite the community members to join the action.

Reviewer 6: ```suggestion
With the lockdowns caused by the COVID-19 pandemic researchers turn to online conferencing. While posing new challenges, this format also brings multiple advantages. We argue that virtual conferences will become part of our regular scientific communication and invite the community members to join the action.
```

Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or have even given one yourself.
The only difference is that it happened online, perhaps as part of a virtual conference.

<!-- #### Online talks were gaining momentum already before, and for a good reason. -->

Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and online events are the only available option.
Online events are also free from most of the expensive and time-consuming aspects of conference organisation: finding a venue, arranging lodging and catering for the participants.
With planning and logistics out of the way, virtual conferences can be organized much faster and can easily respond to the latest scientific developments (see a comparison of typical timelines in the figure).

> Because of being easier to attend and accommodating hundreds of participants, online events are also more inclusive, and reach an even broader audience if the talks are recorded.

Reviewer 6: ```suggestion
All of these changes have the important effect of making attendance and the accommodation of attendants easier, making online events more inclusive.
Finally, such events can easily reach an even broader audience if the talks are recorded and published online.
```
This sentence, particularly its introductory clause, is rather awkward.

Reviewer 4: I like the specificity of the original text. How about: Because they are easier to attend and can accommodate hundreds of participants, online events are also more inclusive…

Author: ```suggestion
Since online events are easier to attend and since they accommodate hundreds of participants, they are also more inclusive.
If the talks are recorded and published, they reach an even broader audience.
```

<!-- #### We decided to develop online conferences and established VSF. -->

The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.
We started exploring the opportunities and challenges of online conferences in late Summer 2019, well before the COVID-19 pandemic.
We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
After several discussions we committed to developing a community platform for online conferences, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).

<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->

Once we started brainstorming how to organize online events, we realized that a lot of usual assumptions about scientific meetings do not apply anymore.
Already the sizes of the events are very different: while a typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract 500 attendees.

> The duration of an online event is limited to a few hours because it needs to fit into the daily routines of participants spread across multiple hour zones, as opposed to the median duration of a conference being a full working week.

Reviewer 4: I suggest reversing the clauses in this sentence to match the flow of the rest of the paragraph.

Author: ```suggestion
On the other hand, the median duration of a traditional conference is a full working week, while the duration of an online event is limited to a few hours because it needs to fit into the daily routines of participants spread across multiple hour zones.
```

> The interactions during the session are also very different: the speaker cannot see the audience and therefore gets very little immediate feedback due to being unable to gauge the audience's reactions.

Reviewer 6: ```suggestion
The interactions during the session are also very different: the speaker cannot see the audience and therefore has no means of immediate feedback.
```

Because mediating the interaction between the audience and the speaker becomes much more important, and because of the larger audience size, the moderators play a more important role in ensuring that everything goes smoothly.
Finally, offline conferences allow ample time for informal discussion between the talks and during coffee breaks: this valuable aspect of scientific meetings needs to be accommodated separately.

> **A typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract 500 attendees.**

<!-- #### We ran a couple of events, and saw that they work well. -->

To help the community members orient themselves in a new setting, we prepared basic guidelines for all online event participants: organizers, moderators, speakers, and audience members.
Once we felt sufficiently confident, we ran an trial workshop with our direct colleagues as both speakers and the audience.

> We used this test run to try out different formats of interaction between the audience and the speaker, check out unexpected technical problems, and collect the impressions from a broader group of colleagues.

Reviewer 6: ```suggestion
We used this test run to try out different formats of interaction between the audience and the speaker, experience unexpected technical problems, and collect the impressions from a broader group of colleagues.
```

Author: A bit confused by this suggestion

Reviewer 2: ```suggestion
We used this test run to try out different formats of interaction between the audience and the speaker, resolve unexpected technical problems, and collect the impressions from a broader group of colleagues.
```

Reviewer 2: how's "resolve"?

> More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.

Reviewer 6: ```suggestion
To our delight, we found that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
```

> More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
> With this experience we were ready to hit the ground running and managed to organize a virtual session only a few days after the cancellation of the APS March Meeting.

Reviewer 5: ```suggestion
More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
With the gained knowledge we were ready to hit the ground running and managed to organize a virtual session only a few days after the cancellation of the APS March Meeting.
```

Author: Experience is broader than knowledge, and is appropriate here.

> With this experience we were ready to hit the ground running and managed to organize a virtual session only a few days after the cancellation of the APS March Meeting.

Reviewer 6: ```suggestion
With this experience we were ready to hit the ground running, and the cancellation of the APS March Meeting was a catalyst for our first virtual session, which we organized in just a few days.
```

Reviewer 2: ```suggestion
With this experience we were ready to hit the ground running. In fact, the cancellation of the APS March Meeting became a catalyst for our first virtual session, which we organized in just a few days.
```

> Now we are continuing to work in a steady pace: we improve the platform and organize a biweekly colloquium series.

Reviewer 6: ```suggestion
Now we are continuing to work in a steady pace: we improve the platform and organize a [biweekly colloquium series](https://virtualscienceforum.org/#/long_range_colloquium).
```

> We also actively invite community initiatives for organizing virtual conferences using our platform, and we are excited to already have the first of such conference applications in progress.

Reviewer 6: ```suggestion
We also actively invite community initiatives to [organize virtual conferences on our platform](https://virtualscienceforum.org/#/announcement), and we are excited to already have the first of such conference applications in progress.
```

Our experience with organizing online conferences and colloquia has shown that they provide a stimulating environment and foster fruitful discussions.

<!-- #### We are confident that we will see an explosive growth of online events. -->

> With more and more people getting used to online meetings and seminars—and with online conferencing software continually improving—we expect that online events become commonplace. 

Reviewer 6: ```suggestion
With more and more people getting used to online meetings and seminars—and with online conferencing software continually improving—we expect that online scientific events will become commonplace. 
```

Our effort towards online conferences was not the first and is certainly no longer one of few.

> Online colloquia and seminars are now organized by many science departments and research institutes.

Reviewer 6: ```suggestion
Such events are now organized by many science departments and research institutes.
```

> Virtual workshops and conferences are also on the rise.

Reviewer 6: I don't think this sentence is necessary.

Author: ```suggestion
```

Emerging organizations like [researchseminars.org](https://researchseminars.org) aggregate the information and distribute announcements in order to help navigate the rapidly growing volume of online events.

<!-- #### You should definitely get involved in the online events, and we are here to help. -->

> **We conclude with a call to action: give virtual conferences a shot!**

Online conferencing is emerging right now because there are no alternatives until the pandemic is over, lockdowns are lifted, and travel is restored.
While our hand is currently forced, we hope that the research community will grow accustomed to this long overdue change and will see the value of virtual conferences in their own right.
This is why we conclude with a call to action: give virtual conferences a shot!
Start with attending the events you like: it has never been easier.
Or perhaps you have an idea for a good event to run?

> In that case, now is your chance: go for it, and know that we are here to help.

Reviewer 6: ```suggestion
Now is your chance: go for it, and know that we are here to help.
```

- Babak Seradjeh (Indiana University)
- Daniel Varjas (Delft University of Technology)

> > Virtual Science Forum is an open, collaborative and volunteer-run effort to accommodate organization of online conferences. Originally initiated by physicist from different backgrounds and locations aiming to provide a service to the research community.

Reviewer 6: ```suggestion
> Virtual Science Forum is an open, collaborative, and volunteer-run effort to facilitate online scientific seminars.  Originally initiated by physicists from different backgrounds and locations, we aim to provide a service and make a positive impact on the research community.
```

Author: Another remark here: we're providing a bio, so let's start with "The authors are some of the members of..."

Also let's expand "different" to be more specific—we likely have space.

Reviewer 5: ```suggestion
> The authors are some of the contributors of Virtual Science Forum, an open, collaborative and volunteer-run effort to facilitate online scientific seminars. Originally initiated by physicists with backgrounds ranging from condensed matter to quantum computation, located all over Europa and United states with the aim to provide a service and and make a positive impact on the research community.
```

Reviewer 6: Updated my suggested change to reflect your remarks.

Reviewer 5: @vfatemi somehow the final
> we aim to make...
doesn't read natural with the 3rd person used at the beginning of the bio.

Reviewer 6: Right. We should pick either 1st or 3rd person overall. I checked recent EPN articles and they are all in third person, so let's do that.

Reviewer 6: >  The authors are members of the Virtual Science Forum, an open, collaborative, and volunteer-run effort to facilitate online scientific seminars. They work at academic institutions spread across North America and Europe and donate their effort to make a positive impact on the research community.

<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->

Once we started brainstorming online events, we realized, that despite those are similar to regular scientific meetings, a lot of our assumptions about conferences do not hold anymore.
For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 

> As a speaker, for example, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 

Reviewer 5: The repetition of *for example* reads odd.

Suggestion:
As a speaker, for instance, gauging the audience's reaction is much harder due to the lack...

Reviewer 2: ```suggestion
For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous. 
As a speaker, for instance, gauging the audience's reaction is much harder due to the lack of visual and audio contact -- most participants will have their audio muted and their video turned off. 
```

Reviewer: Thank for noticing that! I now realized that the transition from the previous sentence to these two is a bit weird. The previous bit talks about assumptions on conferences; Maybe that needs to change?

This in turn makes setting the atmosphere more difficult, not knowing whether potential jokes were a success or not.

> To alleviate some of these challenges and to act as a catalyst between the audience and the speaker, one or multiple moderators are designated to keep an eye out for questions and relay those to the speaker.

Reviewer 5: It is not clear to me how the moderators help the speaker with the jokes. Perhaps make a more direct connection.

Suggestion:
(...) one or multiple moderators are designated to keep an eye out for questions and reactions and transmit them to the speaker.

Reviewer 2: ```suggestion
This in turn makes setting the atmosphere more difficult, not knowing whether potential jokes were a success or not.
To alleviate some of these challenges and to act as a catalyst between the audience and the speaker, one or multiple moderators are designated to keep an eye out for questions and comments and relay those to the speaker.
```
